compiler/integration-tests/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "license": "(MIT OR Apache-2.0)",
   "main": "index.js",
+  "private": true,
```

Comment:
If I don't include this, my publishing script will try to publish the integration tests which we don't want